### PR TITLE
feat: new templates design

### LINKED ISSUE #76: [templates] Reduce size for production error pages
Currently, error pages added to the production bundle have a big impact on server bundle size: 

image

We shall find a super minimal design that also fits for various websites as default error page.

Reference design for Nuxt 2: https://github.com/nuxt/nuxt.js/blob/dev/packages/vue-app/template/views/error.html

### PR DESCRIPTION
**Pull request by atinux**
Resolves #76 

Beautiful work done by @R-mooon, @alexchopin and @Flosciante 🙌 

Preview is on https://templates.ui.nuxtjs.org/

## 404
<table>
<td>Screenshot 2022-03-30 at 13 56 57@2x</td>
<td>Screenshot 2022-03-30 at 13 57 03@2x</td>
</table>

## 500

<table>
<td>Screenshot 2022-03-30 at 13 57 36@2x</td>
<td>Screenshot 2022-03-30 at 13 57 32@2x</td>
</table>

## Error in dev

<table>
<td>Screenshot 2022-03-30 at 13 56 39@2x</td>
<td>Screenshot 2022-03-30 at 13 56 30@2x</td>
</table>

## Welcome

<table>
<td>Screenshot 2022-03-30 at 15 03 55@2x</td>
<td>Screenshot 2022-03-30 at 15 03 51@2x</td>
</table>

## Loading

<table>
<td>Screenshot 2022-03-30 at 15 04 31@2x</td>
<td>Screenshot 2022-03-30 at 15 04 35@2x</td>
</table>

## Animations

https://user-images.githubusercontent.com/904724/160841937-c9a259bd-190d-4508-a467-1ff25ad1b546.mp4

https://user-images.githubusercontent.com/904724/160842033-9fde1910-dd60-4761-93da-6b150030a4d2.mp4

https://user-images.githubusercontent.com/904724/160841890-2525b2d4-a19f-42ae-b6cf-e66f4913ef9d.mp4

### CODE REVIEW

File: packages/templates/templates/welcome/messages.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "title": "Welcome to Nuxt 3!",
-  "documentation": "Documentation",
+  "version": "v3.0.0",
```
Comment (atinux) [inline packages/templates/templates/welcome/messages.json]: Would that be possible to get the version from the running Nuxt @pi0 ?

File: packages/templates/templates/welcome/index.html [hunk 2]
```diff
@@ -36,58 +104,49 @@
...
+            <path d="M16.465 26.8683C17.0456 27.4491 17.7604 27.878 18.5462 28.1169C19.3319 28.3559 20.1644 28.3976 20.9701 28.2385C21.7758 28.0793 22.5299 27.7241 23.1657 27.2043C23.8015 26.6845 24.2995 26.016 24.6157 25.2581C24.9318 24.5001 25.0564 23.6759 24.9784 22.8584C24.9004 22.0408 24.6222 21.2551 24.1684 20.5705C23.7146 19.886 23.0992 19.3238 22.3766 18.9336C21.6539 18.5434 20.8463 18.3373 20.025 18.3333L18.3333 23.3333H15C15 24.6133 15.4883 25.8933 16.465 26.8683Z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+          </svg>
         </div>
+        <p class="mb-2">Remove this welcome page by creating an <a href="https://v3.nuxtjs.org/docs/directory-structure/app" target="_blank" rel="noopener" class="bg-black/10 dark:bg-white/10 rounded font-mono p-1 font-bold">app.vue</a> file.</p>
```
Comment (atinux) [inline packages/templates/templates/welcome/index.html]: Note: I believe it is much better to always recommend to have an `app.vue` (global head, styles, etc.).

**Review by danielroe — COMMENTED** [review]
I love these!

My only question is around the grey boxes on the `app.vue`. They don't seem quite as beautiful as the rest of the pages.

File: packages/templates/templates/welcome/messages.json (hunk 1 above)
Comment (pi0) [reply packages/templates/templates/welcome/messages.json]: Surely we can and it would be a nice addition!

**Comment by atinux** [comment]
Agree with you, updated to be lighter:

Screenshot 2022-03-30 at 15 37 01@2x

**Comment by atinux** [comment]
We can progressively enhance them in next iterations 😊

File: packages/templates/templates/welcome/index.html (hunk 2 above)
Comment (pi0) [reply packages/templates/templates/welcome/index.html]: Small note: We  show this welcome page within default `app.vue` in starter template. They need to remove `<NuxtWelcome>` component.

File: packages/templates/templates/welcome/messages.json (hunk 1 above)
Comment (pi0) [reply packages/templates/templates/welcome/messages.json]: Set it to empty by default and moved to a generic param (top level messages.json)

We need to support process.env for messages and pass env by Nuxt in order to make it working.

File: packages/templates/templates/welcome/index.html (hunk 2 above)
Comment (pi0) [reply packages/templates/templates/welcome/index.html]: https://github.com/nuxt/ui/pull/81/commits/ea4ebaa4127702a4ba5a6bbdb48fb3dd81a765ad

image

**Review by pi0 — APPROVED** [review]

**Pull request merged**